tern/extensions/cve_bin_tool/executor.py

```diff
@@ -25,13 +24,12 @@ class CveBinTool(Executor):
     '''Execute cve-bin-tool'''
     def execute(self, image_obj, redo=False):
         '''Execution should be:
-            cve-bin-tool -u now /path/to/directory
+            cve-bin-tool -x -u now /path/to/directory
         '''
-        command = 'cve-bin-tool -u now'
-        # run the command against the image filesystems
-        if not passthrough.run_on_image(image_obj, command, True, redo=redo):
```

Comment:
Do we still need the `run_on_image` function in `passthrough.py` anymore if we don't call it here? I don't see it being used anywhere else.

Reply:
It's a convenience function for other tools to use. I am not sure if I want to delete it yet.